### PR TITLE
Add a reference to the code gallery.

### PR DESCRIPTION
**Pull request by bangerth**
deal.II recently acquired a parallel N-S solver.

### CODE REVIEW

File: examples/step-57/doc/results.dox [hunk 1]
```diff
@@ -401,4 +401,6 @@ multigrid is needed. See below.
 For larger computations, especially in 3d, it is necessary to implement MPI
 parallel solvers and preconditioners. A good starting point would be step-55,
 which uses algebraic multigrid for the velocity block for the Stokes
-equations.
+equations. Another option would be to take a look at the list of codes
+in the <a href="https://www.dealii.org/code-gallery.html">deal.II code
+gallery</a>, which already contains parallel Navier-Stokes solvers.
```
Comment (tjhei) [inline examples/step-57/doc/results.dox:406]:
```suggestion
gallery</a>, which already contains a parallel Navier-Stokes solver.
```

Comment (bangerth) [reply examples/step-57/doc/results.dox:406]: We actually have two, so plural seemed appropriate :-)

Comment (jppelteret) [reply examples/step-57/doc/results.dox:406]: Yip, there are indeed two that advertise themselves as NS solvers. So I'll consider this answered :-)

**Comment by jppelteret** [comment]
This is just documentation, so we can merge it as is.

**Pull request merged**